aws-rds/README.md

```diff
@@ -24,81 +24,181 @@ Use SignalFx to monitor Amazon Relational Database Service (RDS) via [Amazon Web
   [<img src='./img/dashboard_rds_instance.png' width=200px>](./img/dashboard_rds_instance.png)
 
 ## INSTALLATION
+Choose a deployment method and follow the steps below to encrypt your SignalFx access token, customize the metrics sent to SignalFx, and create and deploy the Lamda function.
 
-### Step 1: Connect to CloudWatch
+### Prerequisites
+Before you begin, you must enable the Enhanced Monitoring option for the RDS instances you want to monitor using this integration. [Click here for instructions on enabling Enhanced Monitoring](http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_Monitoring.OS.html).
 
+* [Deploying through the Serverless Application Repository](#deploying-through-the-serverless-application-repository)
+* [Building from source](#building-from-source)
+* [Metrics collected by this integration](#metric-groups-collected-by-this-integration)
 
-By default, SignalFx will import all CloudWatch metrics that are available in your account. To retrieve metrics for a subset of available services or regions, modify the connection on the Integrations page.
+#### If you are upgrading from version 0.1.0 (Python 2.7) to version 0.2.0 (Python 3.x)
+If you are upgrading to version 0.2.0, ensure that the AWS Lambda handler is set to `enhanced_rds.lambda_script.lambda_handler`.
 
-#### RDS ENHANCED MONITORING
+#### Encrypting your SignalFx access token
+The Lambda function uses your SignalFx access token to send metrics to SignalFx, as an environment variable to the function. While Lambda encrypts all environment variables at rest and decrypts them upon invocation, AWS recommends that all sensitive information such as access tokens be encrypted using a KMS key before function deployment, and decrypted at runtime within the code.
```

Comment:
KMS = Key Management Service. Per the Splunk Style Guide, "Always spell out a full term at first use on every page before you refer to it as an acronym or initialism. "